website/cue/reference/configuration.cue

```diff
@@ -75,6 +75,10 @@ configuration: {
 			description: """
 				If set, Vector will configure the internal metrics system to automatically
 				remove all metrics that have not been updated in the given number of seconds.
+				Note that internal counters that are expired but are later updated will have
+				their values reset to zero.
+				Be careful to set this value high enough to avoid
+				expiring critical but infrequently updated internal counters.
 				This value must be positive.
```

Comment:
```suggestion
				If set, Vector will configure the internal metrics system to automatically
				remove all metrics that have not been updated in the given number of seconds.
				
				Note that internal counters that are expired but are later updated will have
				their values reset to zero.
				Be careful to set this value high enough to avoid
				expiring critical but infrequently updated internal counters.
				
				This value must be positive.
```

Not sure if you want a paragraph break here or not but just a note that you'll need a blank line. I could see doing it something like this to draw more attention to the note.